dll/win32/imm32/imm.c

```diff
@@ -1017,13 +1017,250 @@ LRESULT WINAPI ImmEscapeW(
         return 0;
 }
 
+#ifdef __REACTOS__
+
+#define ROUNDUP4(n) (((n) + 3) & ~3)  /* DWORD alignment */
+
+PCLIENTIMC WINAPI ImmLockClientImc(HIMC hIMC)
```

Comment:
Who should create this PCLIENTIMC object?
What type should it be, probably a win32k handle?

Reply:
Difficult to analyze, it takes time.

Reply:
Locally allocated by `LocalAlloc`.